### PR TITLE
Python workflows

### PR DESCRIPTION
**Pull request by GeekMasher**

### CODE REVIEW

File: .github/workflows/python-linting.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        type: string
+        default: 'ruff'
+      versions:
+        description: 'Python versions to test against'
+        type: string
+        # All Major versions of Python that are currently supported
+        default: '3.9,3.10,3.11,3.12'
+
+jobs:
+  python-verions:
```
Comment (adrienpessu) [inline .github/workflows/python-linting.yml:21]: is it a typo?

File: .github/workflows/python-release.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            echo "No new release found"
+            echo "release=false" >> "$GITHUB_OUTPUT"
+          else
+            echo "New release found"
+            echo "version=$current_version" >> "$GITHUB_OUTPUT"
+            echo "release=true" >> "$GITHUB_OUTPUT"
+          fi
+
+  github-release:
+    uses: advanced-security/reusable-workflows/.github/workflows/release.yml@v0.1.0
```
Comment (felickz) [inline .github/workflows/python-release.yml:60]: FYI `advanced-security/reusable-workflows/.github/workflows/release.yml` doesnt exist in `main` ... looks like only `workflow-releases`

Comment (felickz) [reply .github/workflows/python-release.yml:60]: this is in #36

**Review by felickz — APPROVED** [review]

File: .github/workflows/python-release.yml (hunk 2 above)
Comment (GeekMasher) [reply .github/workflows/python-release.yml]: Updated to `main` for now

File: .github/workflows/python-linting.yml (hunk 1 above)
Comment (GeekMasher) [reply .github/workflows/python-linting.yml]: Updated

**Pull request merged**